foreword.md

```diff
@@ -93,7 +93,7 @@ Being open unlocks many other things.
 
 Local responsibility and democratic accountability are ensured when a public organization implements and maintains their own public code.
 By being open and with a broader contributor base, the software is more secure as it benefits from many eyes spotting potential flaws.
-Many contributors share the maintenance work to keep it functional and modern, which reduces future technical debt.
+Many contributors share the maintenance work to keep it functional and modern, which improves long-term maintainability.
```

Comment:
I wasn't confident that the foreword should change, but reading it now, I agree this is the correct call